ui/app/src/services/content.ts

```diff
@@ -626,7 +632,14 @@ export function moveItem(
           targetParentPath,
           (element) => {
             const item: Element = extractNode(element, targetFieldId, targetIndex);
-            const field: Element = extractNode(element, targetFieldId, removeLastPiece(`${targetIndex}`));
+            let field: Element = extractNode(element, targetFieldId, removeLastPiece(`${targetIndex}`));
+            // If field doesn't exist yet in the document, create it
+            if (!field) {
+              const newField = createElement(originalFieldId);
+              newField.setAttribute('inline', 'true');
```

Comment:
The `inline="true"` attribute is only for the `item` element of **embedded components** of a node selector type **field**. Did you mean `item-list="true"`? Are you creating a field or an item?

Reply:
Yeah I'm creating a field, I got confused about the attribute name. Fixing